### PR TITLE
Add err ID to all errors

### PR DESCRIPTION
**Pull request by Steven-Roberts**

### CODE REVIEW

File: src/+otp/+ascherlineardae/AscherLinearDAEProblem.m [hunk 1]
```diff
@@ -21,7 +21,8 @@ function onSettingsChanged(obj)
         function y = internalSolveExactly(obj, t)
             beta = obj.Parameters.Beta;
             if ~isequal(obj.Y0, [1; beta])
-                error('An exact solution is unavailable for this initial condition');
+                error('OTP:noExactSolution', ...
```
Comment (AndreyAPopov) [inline src/+otp/+ascherlineardae/AscherLinearDAEProblem.m:24]: It seems like there is no consistent casing, and lower case 'no' has precedent in matlab. Weird, but this seems correct.

File: src/+otp/+utils/+movie/+recorder/NullVideoRecorder.m [hunk 2]
```diff
@@ -26,7 +26,7 @@ function stop(~)
         
         function h = play(~)
             h = 'Video not saved for playback';
-            error(h);
+            error('OTP:movieNotSaved', h);
```
Comment (AndreyAPopov) [inline src/+otp/+utils/+movie/+recorder/NullVideoRecorder.m:29]: maybe do 'OTP:noVideoSaved'? or 'OTP:videoNotSaved'? We use the term video and not movie.

File: src/+otp/+utils/+movie/CometMovie.m [hunk 3]
```diff
@@ -54,19 +50,16 @@ function drawFrameAxes(obj, ~, gObjects, state)
...
         function x = getXPoints(obj, cometIdx, state)
             x = otp.utils.compatibility.abstract(obj, cometIdx, state);
         end
         
         function y = getYPoints(obj, cometIdx, state)
             y = otp.utils.compatibility.abstract(obj, cometIdx, state);           
         end
+        
+        function z = getZPoints(obj, cometIdx, state)
+            z = otp.utils.compatibility.abstract(obj, cometIdx, state);
```
Comment (AndreyAPopov) [inline src/+otp/+utils/+movie/CometMovie.m:62]: Does 3d work now?

**Comment by AndreyAPopov** [comment]
Everything else other than some minor comments looks good

File: src/+otp/+utils/+movie/+recorder/NullVideoRecorder.m (hunk 2 above)
Comment (Steven-Roberts) [reply src/+otp/+utils/+movie/+recorder/NullVideoRecorder.m:29]: Fixed in other PR so movie is always used

**Pull request merged**